package.json

```diff
@@ -117,8 +117,10 @@
     "@babel/cli": "^7.8.4",
```

Comment:
The body parers are here for the test suites.  Even though we discovered that out `koa` and `fastify` instrumentation does not capture the body correly (i.e doesn't need to be tested), the test harness I came up with becomes awkward if this middleware isn't there.  The was a waffling decision -- happy to reconsider it.